### DISCUSSION TITLE
Why QSimOptions is throwing error when use_sampler disable_gpu are used ?

### DISCUSSION THREAD

**Discussion by ShashiQubit** [NVIDIA/cuQuantum#17] [Q&A] [answered]
`TypeError                                 Traceback (most recent call last)
Cell In [9], line 2
      1 ngpus = 1
----> 2 qsim_options = qsimcirq.QSimOptions(
      3         max_fused_gate_size = 2
      4         , cpu_threads = 1
      5         , gpu_mode = ngpus
      6         , use_sampler = True
      7         , disable_gpu = False
      8     )
      9 qsim_simulator = qsimcirq.QSimSimulator(qsim_options)

TypeError: __init__() got an unexpected keyword argument 'use_sampler'`

It's working fine when these 2 options are removed. I am using windows wsl to run it locally using conda installation.
The documentation clearly has these options. [Link to QSimOptions](https://docs.nvidia.com/cuda/cuquantum/appliance/cirq.html)

**Comment by leofang** [comment, marked as answer]
Because the documentation is specific to our cuQuantum Appliance docker container. The cited option is not available in the public qsimcirq.
